multisig/proposals/arbitrumsepolia/vip-005/index.ts

```diff
@@ -0,0 +1,44 @@
+import { makeProposal } from "../../../../src/utils";
+
+const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
```

Comment:
Should maintain consistency 
```suggestion
export const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
```

Reply:
Got it.
Done https://github.com/VenusProtocol/vips/pull/251/commits/7b37bc688ac4e69a764173a41718c816e34b2be4